### PR TITLE
[DEMO]: remove form from components

### PR DESCRIPTION
**Pull request by BennyAlex**
there is no form route so I removed it

### CODE REVIEW

**Comment by miguelcobain** [comment]
We want it to be a Forms page for ember-paper. So, we want the ending S

**Comment by BennyAlex** [comment]
But where it is the plural? 
And do you want to have a Forms Demo and a Forms Page?

File: tests/dummy/app/templates/application.hbs [hunk 1]
```diff
@@ -41,7 +40,7 @@
              {{#submenu-item active=(is-active "demo.virtual-repeat" currentRouteName) onClick=(transition-to "demo.virtual-repeat")}}Virtual Repeat{{/submenu-item}}
           {{/liquid-if}}
         </div>
-        {{#menu-item active=(is-active "forms" currentRouteName) onClick=(transition-to "forms")}}Forms{{/menu-item}}
+        {{#menu-item active=(is-active "forms" currentRouteName) onClick=(transition-to "forms")}}Form{{/menu-item}}
```
Comment (miguelcobain) [inline tests/dummy/app/templates/application.hbs]: You've replaced `Forms` with `Form`. What is the reason for that?
We thought of `Forms` to be a general purpose sectionabout "Forms" in ember-paper, not just about paper-form.

**Comment by miguelcobain** [comment]
Thanks!

**Comment by BennyAlex** [comment]
I just renamed it because the other navbar item names are singular

**Comment by BennyAlex** [comment]
Theres is also a bug on the forms site, showing "comonents >" before "form"
image

**Comment by miguelcobain** [comment]
Nice catch. :)
Can you PR that as well, please?

**Comment by BennyAlex** [comment]
As far i can see the isDemo attribute is already false. 
So probably ypu have to update the demo page?

**Comment by miguelcobain** [comment]
That's right. :face_with_head_bandage:

**Comment by BennyAlex** [comment]
I dont know exactly how the demo page works, but would it be possible to use the master branch for the page? so you dont need to update it manually

**Comment by DanChadwick** [comment]
@BennyAlex Seen "Building and deploying the dummy app to github" in [contributing.md](https://github.com/miguelcobain/ember-paper/blob/master/CONTRIBUTING.md)

**Comment by BennyAlex** [comment]
@DanChadwick I dont want to build or deploy this, in the end I will break something. I just thought if it is possible to do this automatic, based on the master branch

**Pull request merged**